src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1972,1022 +2130,835 @@ void UpdateZoneSizing(EnergyPlusData &state, DataGlobalConstants::CallIndicator
+                        static constexpr std::string_view ZSizeFmt21("{}{:12.6E}{}{:12.6E}{}{:12.6E}{}{:12.6E}");
+                        print(state.files.zsz,
+                              ZSizeFmt21,
+                              state.dataSize->SizingFileColSep,
+                              state.dataSize->CalcFinalZoneSizing(I).HeatLoadSeq(TimeStepIndex),
+                              state.dataSize->SizingFileColSep,
+                              state.dataSize->CalcFinalZoneSizing(I).CoolLoadSeq(TimeStepIndex),
+                              state.dataSize->SizingFileColSep,
+                              state.dataSize->CalcFinalZoneSizing(I).HeatFlowSeq(TimeStepIndex),
+                              state.dataSize->SizingFileColSep,
+                              state.dataSize->CalcFinalZoneSizing(I).CoolFlowSeq(TimeStepIndex));
                     }
+                    print(state.files.zsz, "\n");
                 }
             }
+            print(state.files.zsz, "Peak");
+
+            for (I = 1; I <= state.dataGlobal->NumOfZones; ++I) {
+                if (!state.dataZoneEquip->ZoneEquipConfig(I).IsControlled) continue;
+
+                static constexpr std::string_view ZSizeFmt31("{}{:12.6E}{}{:12.6E}{}{:12.6E}{}{:12.6E}");
+                print(state.files.zsz,
+                      ZSizeFmt31,
+                      state.dataSize->SizingFileColSep,
+                      state.dataSize->CalcFinalZoneSizing(I).DesHeatLoad,
+                      state.dataSize->SizingFileColSep,
+                      state.dataSize->CalcFinalZoneSizing(I).DesCoolLoad,
+                      state.dataSize->SizingFileColSep,
+                      state.dataSize->CalcFinalZoneSizing(I).DesHeatMassFlow,
+                      state.dataSize->SizingFileColSep,
+                      state.dataSize->CalcFinalZoneSizing(I).DesCoolMassFlow);
+            }
+            print(state.files.zsz, "\n");
+
+            print(state.files.zsz, "\nPeak Vol Flow (m3/s)");
+            for (I = 1; I <= state.dataGlobal->NumOfZones; ++I) {
+                if (!state.dataZoneEquip->ZoneEquipConfig(I).IsControlled) continue;
+                static constexpr std::string_view ZSizeFmt41("{}{}{}{:12.6E}{}{:12.6E}");
+                print(state.files.zsz,
+                      ZSizeFmt41,
+                      state.dataSize->SizingFileColSep,
+                      state.dataSize->SizingFileColSep,
+                      state.dataSize->SizingFileColSep,
+                      state.dataSize->CalcFinalZoneSizing(I).DesHeatVolFlow,
+                      state.dataSize->SizingFileColSep,
+                      state.dataSize->CalcFinalZoneSizing(I).DesCoolVolFlow);
+            }
+            print(state.files.zsz, "\n");
+            state.files.zsz.close();
+        }
 
+        // Move data from Calc arrays to user modified arrays
+
+        for (std::size_t i = 0; i < state.dataSize->ZoneSizing.size(); ++i) {
```

Comment:
would `state.dataSize->ZoneSizing[i] = state.dataSize->CalcZoneSizing[i]` not work?